cinch/bin/wrappers.py

```diff
@@ -13,30 +15,115 @@
 
 def call_ansible(inventory, *args):
     """
-    Wraps a call out to the ansible-playbook executable, passing it the cinch site.yml
-    file that kicks off this playbook.
+    Wraps a call out to the ansible-playbook executable, passing it the cinch
+    site.yml file that kicks off this playbook.
 
     :param inventory: The Ansible inventory file to pass through
-    :param args: An array of other command-line arguments to ansible-playbook to pass
-    :return: The exit code returned from ansible-playbook, or 255 if errors come from elsewhere
+    :param args: An array of other command-line arguments to ansible-playbook
+    to pass
+    :return: The exit code returned from ansible-playbook, or 255 if errors
+    come from elsewhere
     """
     # Construct the arguments to pass to Ansible by munging the arguments
     # provided to this method
     ansible_args = [
         path.join(BASE, 'site.yml'),
         '-i', inventory,
-        '-v'
+        '-v',
+        '--ssh-common-args=-o StrictHostKeyChecking=no'
     ]
     ansible_args.extend(args)
     ansible = local['ansible-playbook']
+    command_handler(ansible, ansible_args)
+
+
+def call_linchpin(work_dir, *args):
+    """
+    Wraps a call out to the linchpin executable, and then kicks off a cinch
+    Ansible playbook if necessary.
+
+    :param work_dir: The linch-pin working directory that contains a PinFile
+    and associated configuration files
+    :param args: An array of other command-line arguments to linchpin to pass
+    :return: The exit code returned from linchpin, or 255 if errors come from
+    elsewhere
+    """
+    # Construct the arguments to pass to linchpin by munging the arguments
+    # provided to this method
+    linchpin_args = list()
+    linchpin_args.extend(args)
```

Comment:
This code has been removed since we are only supporting a single linch-pin argument.